ember-toucan-core/src/components/form/controls/textarea.ts

```diff
@@ -0,0 +1,18 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export interface ToucanFormTextareaControlComponentSignature {
+  Element: HTMLTextAreaElement;
+  Args: {
+    isDisabled?: boolean;
+    onChange?: (value: string, e: Event | InputEvent) => void;
+    value?: string;
+  };
+}
+
+export default class ToucanFormTextareaControlComponent extends Component<ToucanFormTextareaControlComponentSignature> {
+  @action
+  handleInput(e: Event | InputEvent): void {
+    this.args.onChange?.((e.target as HTMLTextAreaElement)?.value, e);
```

Comment:
instead of having `as` you can assert on the element:

```
    assert('Expected HTMLTextAreaElement', e.target instanceof HTMLTextareaElement);
```

Reply:
Interesting!  Thanks - went ahead and applied!